youtube_dl/extractor/qqmusic.py

```diff
@@ -119,6 +119,12 @@ def _real_extract(self, url):
             'upload_date': publish_time,
             'creator': singer,
             'description': lrc_content,
+            'subtitles': {
+                'zh-CN': [{
```

Comment:
@dstftw Yes, there're lyrics in other languages. I don't know what to write actually. zh-CN might be the best option if it has to be a valid language code. Please advise.